lib/ecto/changeset.ex

```diff
@@ -454,6 +454,59 @@ defmodule Ecto.Changeset do
     end)
   end
 
+  @doc """
+  Returns true if a field was changed in a changeset
+
+  ## Options
+
+    * `:to` - Check if the field was changed to a specific value
+    * `:from` - Check if the field was changed from a specific value
+
+  ## Examples
+
+      iex> post = %Post{title: "Foo", body: "Old"}
+      iex> changeset = change(post, %{title: "New title", body: "Old"})
+
+      iex> changed?(changeset, :body)
+      false
+
+      iex> changed?(changeset, :title)
+      true
+
+      iex> changed?(changeset, :title, to: "NEW TITLE")
+      false
+  """
+  @spec changed?(t, atom, Keyword.t) :: boolean
+  def changed?(%Changeset{} = changeset, field, opts \\ []) when is_atom(field) do
+    case fetch_change(changeset, field) do
+      {:ok, new_value} ->
+        changed_from?(changeset, field, opts) and changed_to?(new_value, opts)
```

Comment:
```suggestion
        Enum.all?(opts, fn
          {:from, from} -> Map.get(changeset.data, field) == from
          {:to, to} -> new_value == to
          other -> raise ArgumentError, "unknown option #{inspect(other)}"
        end)
```

Reply:
That's clever. I was thinking that there should be a better way to do it :)